packages/lib/delivery/src/notifications/utils.ts

```diff
@@ -0,0 +1,38 @@
+import { DeliveryInformation } from '@dm3-org/dm3-lib-messaging';
```

Comment:
It think it would be more concise to name the file fetchEmailTemplates or similiar instead of utils.ts

Reply:
I thought to have utility functions like fetchEmailTemplates() and generateOtp() function in this file that will be used for notification purpose. So that it would not end up in creating multiple files for small functions.

Reply:
I prefer to have two different files because you can find them better via file search and they are more visible in the file tree. Whereas utils.ts barely gives any information about the functions it contains.
However, it's not a dealbreaker for me. If you prefer this kind of naming pattern it's fine for me too.

Reply:
No, I will follow the same structure that you have been following so there will be consistency. I will create separate files for both functions.